recipe/meta.yaml

```diff
@@ -39,6 +39,19 @@ outputs:
         - sysroot_{{ target_platform }}
       run:
         - sysroot_{{ target_platform }}
+      run_constrained:
+        - gcc_impl_{{ target_platform }} !=7.2.0
+        - gcc_impl_{{ target_platform }} !=7.3.0
+        - gcc_impl_{{ target_platform }} !=8.2.0
+        - gcc_impl_{{ target_platform }} !=5.4.0
+        - gxx_impl_{{ target_platform }} !=7.2.0
+        - gxx_impl_{{ target_platform }} !=7.3.0
+        - gxx_impl_{{ target_platform }} !=8.2.0
+        - gxx_impl_{{ target_platform }} !=5.4.0
+        - gfortran_impl_{{ target_platform }} !=7.2.0
+        - gfortran_impl_{{ target_platform }} !=7.3.0
+        - gfortran_impl_{{ target_platform }} !=8.2.0
+        - gfortran_impl_{{ target_platform }} !=5.4.0
```

Comment:
Don’t we need to limit this to versions one that platform? Otherwise how will we build 8.2 on Linux-64 for example.

Reply:
We won't. We'll go to `8.4.0`